src/super_gradients/training/qat_trainer/qat_trainer.py

```diff
@@ -89,6 +90,29 @@ def train_from_config(cls, cfg: Union[DictConfig, dict]) -> Tuple[nn.Module, Tup
             dataloader_params=cfg.dataset_params.val_dataloader_params.copy(),
         )
 
+        if "calib_dataloader" in cfg:
+            calib_dataloader_name = get_param(cfg, "calib_dataloader")
+            calib_dataloader_params = cfg.dataset_params.calib_dataloader_params.copy()
```

Comment:
maybe copy.deepcopy() instead? Just to play safe and avoid any change of having a shallow copy?